cmd/realm/triggers.go

```diff
@@ -69,111 +71,351 @@ var triggersDeleteCmd = &cobra.Command{
 	Aliases: []string{"del"},
 }
 
+var triggersSaveCmd = &cobra.Command{
+	Use:   "save <destination-path>",
+	Short: "Save triggers to a local folder",
+	Long: `Save each trigger in a realm to a local folder. Each trigger will
+be saved in a dedicated file whose name will be in the form '<trigger_name>.json'.
+When no destination path is set, triggers will be saved in the current working directory.
+This command does not support the --to-curl flag.`,
+	Example: `  astartectl realm-management triggers save`,
+	Args:    cobra.MaximumNArgs(1),
+	RunE:    triggersSaveF,
+}
+
+var triggersSyncCmd = &cobra.Command{
+	Use:   "sync <trigger_files> [...]",
+	Short: "Synchronize triggers",
+	Long: `Synchronize triggers in the realm with the given files.
+All given files will be parsed, and only new triggers will be installed in the
+realm, depending on the realm's state. In order to force triggers update, use --force flag`,
+	Example: `  astartectl realm-management triggers sync triggers/*.json`,
+	Args:    cobra.MinimumNArgs(1),
+	RunE:    triggersSyncF,
+}
+
 func init() {
-	RealmManagementCmd.AddCommand(triggersCmd)
 
+	RealmManagementCmd.AddCommand(triggersCmd)
+	triggersSyncCmd.Flags().Lookup("force").NoOptDefVal = "false"
```

Comment:
Also, if you set the `NoOptDefVal` here rather than in the sync function, I think that the lookup is redundant: something like this could work, too:
```go

	tiggersSyncCmd.PersistentFlags().BoolVar(...)
	tiggersSyncCmd.Flag("force").NoOptDefVal = "true"
```